client/driver/rkt.go

```diff
@@ -188,20 +205,34 @@ func (d *RktDriver) Start(ctx *ExecContext, task *structs.Task) (DriverHandle, e
 	// Add CPU isolator
 	cmdArgs = append(cmdArgs, fmt.Sprintf("--cpu=%vm", int64(task.Resources.CPU)))
 
-	// Add DNS servers
-	for _, ip := range driverConfig.DNSServers {
-		if err := net.ParseIP(ip); err == nil {
-			msg := fmt.Errorf("invalid ip address for container dns server %q", ip)
-			d.logger.Printf("[DEBUG] driver.rkt: %v", msg)
-			return nil, msg
-		} else {
-			cmdArgs = append(cmdArgs, fmt.Sprintf("--dns=%s", ip))
-		}
+	// Fingerprint
+	node := &structs.Node{
+		Attributes: make(map[string]string),
 	}
+	_, err := d.Fingerprint(&config.Config{}, node)
```

Comment:
@achanda What are you trying to do here? This is going to crash since you are calling the Fingerprint recursively here.